mteb/models/colbert_models.py

```diff
@@ -54,6 +58,36 @@ def __init__(
             self.model.prompts = model_prompts
         self.model_prompts = self.validate_task_to_prompt_name(model_prompts)
 
+    def create_index(self) -> None:
+        """Creates an index for the model."""
+        from pylate import indexes
+        from pylate import retrieve as colbert_retrieve
+
+        uuid_str = str(uuid.uuid4())
+        self.index = indexes.PLAID(
+            index_folder="pylate-index/" + uuid_str,
```

Comment:
I think this won't work on windows
```suggestion
            index_folder=os.path.join("pylate-index", uuid_str),
```